vim.exe.stackdump

```diff
@@ -0,0 +1,19 @@
+Stack trace:
```

Comment:
@glcj a apache header is missing for this file ;)
https://www.apache.org/legal/src-headers.html

Reply:
Ready for the next commit, thanks.